static/templates/right_sidebar.hbs

```diff
@@ -31,7 +31,7 @@
             </a>
             <div class="only-visible-for-spectators">
                 <div class="realm-description">
-                    <p>{{ realm_description }}</p>
+                    <div class="rendered_markdown">{{{ realm_description }}}</div>
```

Comment:
This should be `{{rendered_markdown realm_description}}` -- that does the appropriate HTML escaping.

@andersk should we add a lint rule banning new instances of `{{{` (we can do exclude lines for the existing uses)